package.json

```diff
@@ -78,8 +79,10 @@
     "react-error-boundary": "^3.1.4",
     "react-i18next": "^12.0.0",
     "react-router-dom": "^6.3.0",
+    "storybook-addon-state": "^1.0.3",
     "typescript": "^4.4.2",
-    "vite": "^3.0.0"
+    "vite": "^3.0.0",
+    "yarn": "^1.22.19"
```

Comment:
Why is yarn added? The binary in `.yarn/releases` should be used, so that we're all using the same version

Reply:
No idea, I'll remove it. Maybe I had a typo in my `yarn add` command for `storybook-addon-state`